deploy/install.sh

```diff
@@ -36,14 +36,23 @@ install_docker() {
     sudo ${package_manager} -y update --quiet
     echo "Installing docker"
     sudo ${package_manager} -y install docker-ce docker-ce-cli containerd.io --quiet
-   
-    if [ ! -f /usr/bin/docker-compose ];then
-        echo "Installing docker-compose"
-        sudo curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-        sudo chmod +x /usr/local/bin/docker-compose
-        sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
-    fi
+}
 
+install_docker_compose() {
+    if [ $package_manager == "apt-get" -o $package_manager == "yum" ];then
+        if [ ! -f /usr/bin/docker-compose ];then
+            echo "Installing docker-compose..."
+            sudo curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+            sudo chmod +x /usr/local/bin/docker-compose
+            sudo ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+            echo "docker-compose installed!"
+        fi
+    else
+        echo "docker-compose not found! Please install docker-compose first and then continue with this installation."
```

Comment:
```suggestion
        echo "+++++++++++ IMPORTANT READ ++++++++++++++++++++++"
        echo "docker-compose not found! Please install docker-compose first and then continue with this installation."
```

Suggest adding a section in order to help the user pin-point the error quickly.